runtimes/pythonrt/runner/main.py

```diff
@@ -125,7 +127,7 @@ def ShouldKeepRunning(self, intial_delay=10, period=10):
             time.sleep(period)
 
         log.error("could not verify if should keep running, killing self")
-        os._exit(1)
+        self.server.stop(SERVER_GRACE)
```

Comment:
Is it intentional that the proc's return value is no longer 1? Also, I'm assuming there's a reason you didn't use `sys.exit()`?

(Same in line 307)

Reply:
There's no need, server stop will cause the ` server.wait_for_termination()` to exit which will exit the process.